LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (C) 2014-2020      Mellanox Technologies Ltd. All rights reserved.
+Copyright (c) 2018-2020      NVIDIA CORPORATION. All rights reserved.
```

Comment:
As far I know, we don't put nvidia copyright and still use Mellanox only.

Reply:
@sashakot there are some changes in the source  from NVIDIA team before both companies are merged.